examples/hand_pose_estimation/hand_keypoints_loader.py

```diff
@@ -0,0 +1,58 @@
+import glob
+import pickle
+
+from paz.abstract import Loader
+from paz.backend.image.opencv_image import load_image, resize_image
+
+
+class HandDataset(Loader):
+    def __init__(self, path, split='training', image_size=(256, 256, 3)):
```

Comment:
the split variable should be 'train' or 'val' or 'test' ideally you can use the paz.processors.TRAIN, paz.processors.VAL, paz.processors.TEST